src/index.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable no-unused-vars */
+
 import _ from 'lodash';
```

Comment:
**Kindly work on these requested changes:**
- index.html main contain only root element.
- Use ES6 modules to write modular JavaScript with proper ES6 syntax.
usage of ` arrow functions`  instead of the ` function ` keyword
usage of  ` import ` and ` export ` instead of ` require ` or ` module.exports`

proper usage of ` const ` and ` let ` keywords.

this information it's described in your project requirements:
Using the modules of ES6 and Imports and exports.


- **Use ES6 modules, with import and export.**

Reply:
And as you can see, I used import not require in src/index.js